lib/puma/single.rb

```diff
@@ -17,7 +17,7 @@ class Single < Runner
     def stats
       {
         started_at: utc_iso8601(@started_at)
-      }.merge(@server.stats).merge(super)
```

Comment:
I'm such an old rubyist that safe navigation still feels "new" to me. I had to look up what ruby version this was added it to make sure we support it 💀